app.test.js

```diff
@@ -125,6 +127,72 @@ describe("/api/articles/:article_id", () => {
         });
     });
   });
+
+  describe("PATCH an article", () => {
+    test("200: it should be able to increase votes on an article by ID", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: 1 })
+        .expect(200)
+        .then((response) => {
+          const { article } = response.body;
+
+          expect(article).toHaveProperty("article_id", 1);
+          expect(typeof article.article_id).toBe("number");
+          expect(article).toHaveProperty("votes", 101);
+          expect(article).toHaveProperty("title", expect.any(String));
+          expect(article).toHaveProperty("topic", expect.any(String));
+          expect(article).toHaveProperty("author", expect.any(String));
+          expect(article).toHaveProperty("body", expect.any(String));
+          expect(article).toHaveProperty("created_at", expect.any(String));
+        });
+    });
+    test("200: it should be able to reduce votes on an article by ID", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes: -100 })
+        .expect(200)
+        .then((response) => {
+          const { article } = response.body;
+
+          expect(article).toHaveProperty("article_id", 1);
+          expect(typeof article.article_id).toBe("number");
+          expect(article).toHaveProperty("votes", 0);
+          expect(article).toHaveProperty("title", expect.any(String));
+          expect(article).toHaveProperty("topic", expect.any(String));
+          expect(article).toHaveProperty("author", expect.any(String));
+          expect(article).toHaveProperty("body", expect.any(String));
+          expect(article).toHaveProperty("created_at", expect.any(String));
+        });
+    });
+    test("400: responds with error when invalid id is input", () => {
+      return request(app)
+        .patch("/api/articles/banana")
+        .send({ inc_votes: -100 })
+        .expect(400)
+        .then((response) => {
+          expect(response.body.msg).toBe("Bad request");
+        });
+    });
+    test("GET 404: sends an appropriate and error message when given a valid but non-existent id", () => {
+      return request(app)
+        .patch("/api/articles/9000")
+        .send({ inc_votes: 100 })
+        .expect(404)
+        .then((response) => {
+          expect(response.body.msg).toBe("Article does not exist");
+        });
+    });
+    test("400: responds with error when no votes are are included", () => {
+      return request(app)
+        .patch("/api/articles/1")
+        .send({})
+        .expect(400)
+        .then((response) => {
+          expect(response.body.msg).toBe("Bad request");
+        });
+    });
```

Comment:
Great testing so far! Could you just add one more please? What do we think might happen if we're sending inc_votes and it's not a number? I.e. {inc_votes: "banana"} ?